<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>368</width>
     <height>624</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>823</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget{background-color: rgb(25,25, 40);}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: orange;
font: 12pt &quot;Arial&quot;;
border-radius: 1px;
</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>15</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/icons/rocket_48x48.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMensaje">
        <property name="styleSheet">
         <string notr="true">color: rgb(231, 231, 231);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>35</number>
        </property>
        <property name="rightMargin">
         <number>59</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtUsuario">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    color: rgb(231, 231, 231);
    font: 15pt &quot;Verdana&quot;;
    border: None;
    border-bottom-color: white;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(25, 25, 40);
    selection-background-color: darkgray;
}

QLineEdit:focus {
    border: 2px solid #4d4dff; /* Cambia el color del borde cuando el QLineEdit est치 enfocado */
}

QLineEdit:hover {
    border-bottom-color: #4d4dff; /* Cambia el color del borde inferior cuando el mouse est치 sobre el QLineEdit */
}

</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="styleSheet">
           <string notr="true">border: 2px solid white;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="txtPassword">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    color: rgb(231, 231, 231);
    font: 15pt &quot;Verdana&quot;;
    border: None;
    border-bottom-color: white;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(25, 25, 40);
    selection-background-color: darkgray;
}

QLineEdit:focus {
    border: 2px solid #4d4dff; /* Cambia el color del borde cuando el QLineEdit est치 enfocado */
}

QLineEdit:hover {
    border-bottom-color: #4d4dff; /* Cambia el color del borde inferior cuando el mouse est치 sobre el QLineEdit */
}
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="styleSheet">
           <string notr="true">border: 2px solid green;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="btnAcceder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: rgb(20, 200, 220);
    font: 87 17pt &quot;Arial Black&quot;;
    border: 2px solid green;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}

QPushButton:hover {
    color: rgb(20, 200, 220);
    background-color: rgba(20, 200, 220, 100);
}

QPushButton:pressed {
    background-color: rgba(20, 200, 220, 50);
}


</string>
          </property>
          <property name="text">
           <string>Acceder</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btnRegistrarUsuario">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: rgb(20, 200, 220);
    font: 87 17pt &quot;Arial Black&quot;;
    border: 2px solid green;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}

QPushButton:hover {
    color: rgb(20, 200, 220);
    background-color: rgba(20, 200, 220, 100);
}

QPushButton:pressed {
    background-color: rgba(20, 200, 220, 50);
}
</string>
          </property>
          <property name="text">
           <string>Registrarse</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../lal.qrc"/>
 </resources>
 <connections/>
</ui>
